gpii/node_modules/nativeSettingsHandler/src/nativeSettingsHandler.js

```diff
@@ -230,6 +230,27 @@ windows.nativeSettingsHandler.SetVolume = function (num) {
     });
 };
 
+/**
+ * Gets the current system audio channel mute state, calling the native executable 'VolumeControl.exe' with
+ * the proper payload.
+ *
+ * @return {Promise} A promise that resolves on success or holds a object with the error information.
+ */
+windows.nativeSettingsHandler.GetMute = function () {
+    return windows.nativeSettingsHandler.VolumeHandler("Get", "Mute");
+};
+
+/**
+ * Sets the current system audio channel mute state, calling the native executable 'VolumeControl.exe' with
+ * the proper payload.
+ *
+ * @param {Number} num The value to set as the current system audio channel mute state. Value can be '0' or '1'.
+ * @return {Promise} A promise that resolves on success or holds a object with the error information.
+ */
+windows.nativeSettingsHandler.SetMute = function (num) {
+    return windows.nativeSettingsHandler.VolumeHandler("Set", "Mute", num);
```

Comment:
Instead of this, get the current mute state and if it doesn't match the desired state, then send `APPCOMMAND_VOLUME_MUTE` like in line 227.

This is so there's a visual indication of the change (the person could be deaf, or in an environment where sound isn't nice).

Keep the native code, in the future we might want a "no-ui" option.

Reply:
I understand, so the visual feedback we are looking for is the actual Windows interface popup that is displayed when volume is changes true?

Reply:
yes